File: Member/YangSiyuan/QingGuo_requests/test/orderSubmit.py
from code import Common

from code.Sendhttp import SendHttp
#提交订单

class orderSubmit():
    def setUp(self):
        self.url="/fgadmin/orders/submit"
    def submit_test(self):
        user = {
            "phoneArea": "86",
            "phoneNumber": "200000000096",
            "password": "netease123"
        }
        detial={"skuIds":"3",
                "skuNumbers":"1",
                "stockIds":"74966313",
                "receiverName":"杨san",
                "cellPhone":"12345678909",
                "addressDetail":"河北师范",
                "province":"河北省",
                "city":"石家庄",
                "area":"裕华区",
                "voiceStatus":0,
                "needInvoice":0,
                "invoiceHead":"",
                "transportFee":6,
                "logisticsCompanyId":1,
                "accessSource":"noSource",
                "accessDevice":0}
        result=SendHttp().sent_post_bycookies(self.url,Common.getcookies(user),detial)
        print(result)
        self.assertEqual(result['code'], 200)

File: Member/YangSiyuan/QingGuo_httpclient/src/QGTest/OrderSubmit.java
package QGTest;

import java.util.HashMap;
import java.util.Map;

import org.testng.annotations.Test;

import Core.HttpDriver;
 
public class OrderSubmit {
	
	String url="/fgadmin/orders/submit";//url=http://study-perf.qa.netease.com/fgadmin/orders/submit
	public String odersubmit(String skuIds,String skuNumbers,String stockIds) {
		Map<String,Object>submit=new HashMap<String,Object>();
		submit.put("skuIds", skuIds);
		submit.put("skuNumbers", skuNumbers);
		submit.put("stockIds", stockIds);
		String result=HttpDriver.doPostByForm(url, submit);
		return result;
		}
	@Test
	public void submitTest() {
		String res=this.odersubmit("2,3", "1,1", "74966312,74966313");
		System.out.println(res);
	} 
}
